engine/admin/index.md

```diff
@@ -116,22 +116,21 @@ manager. For Ubuntu 15.04 and higher, refer to [control and configure Docker
 with systemd](systemd.md).
 
 You configure the `docker` daemon in the `/etc/default/docker` file on your
-system. You do this by specifying values in a `DOCKER_OPTS` variable.
+system, or `/etc/systemd/docker` if your Linux distribution uses `systemd`. You
+do this by specifying values in a `DOCKER_OPTS` variable.
```

Comment:
This sentence about `DOCKER_OPTS` was always confusing to me because it can also be an environment variable.
Given that the `etc/default/docker` file has comments that teach you about the syntax, and there are other variables in that file, like `DOCKERD`, can we remove the sentence completely?